Add plugin: Paste From History

Karakaz opened the pull request:
# I am submitting a new Community Plugin

## Repo URL

Link to my plugin: https://github.com/karakaz/obsidian-paste-from-history

## Release Checklist
- [x] I have tested the plugin on
  - [x]  Windows
  - [ ]  macOS
  - [x]  Linux
  - [ ]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [x] My GitHub release contains all required files
  - [x] `main.js`
  - [x] `manifest.json`
  - [x] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the developer policies at https://docs.obsidian.md/Developer+policies, and have assessed my plugins's adherence to these policies.
- [x] I have read the tips in https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

joethei commented:
Is there a reason you decided on making this a plugin, instead of using the clipboard history provided by the OS?

["description": "Adds the ability to paste text from the editor's clipboard history.",](https://github.com/Karakaz/obsidian-paste-from-history/blob/bbc62588876ae7cbeb7148d22a891c1e55ea6830/manifest.json#L6)
 Good descriptions often start with an action statement.
We have published a guide for plugin descriptions to follow here: https://docs.obsidian.md/Plugins/Releasing/Submission+requirements+for+plugins#Keep+plugin+descriptions+short+and+simple.

[.setTitle("Clipboard History")](https://github.com/Karakaz/obsidian-paste-from-history/blob/bbc62588876ae7cbeb7148d22a891c1e55ea6830/src/commands/PasteFromClipboardHistoryCommand.ts#L35)
[Use sentence case in UI](https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines#Use+sentence+case+in+UI)

Karakaz commented:
The main motivation is that I was used to this functionality from IntelliJ and haven't been using the OS integrated ones personally. During the Christmas holidays I was browsing for a plugin like this in the community plugins but couldn't find one and thought it would be a fun holiday project to do.

I've been working on a feature branch that have just been finalized and would be nice to have for the release of the plugin. Can I merge that into master and build a new minor version including addressing your points?
